optics-core/src/Optics/Review.hs

```diff
@@ -25,6 +25,7 @@ module Optics.Review
 
   -- * Subtyping
   , A_Review
+  -- | <<Review.png Review in the optics hierarchy>>
   , toReview
```

Comment:
Looks like `toReview` crept through when we culled the other similar functions?

Reply:
Yeah, let's remove it.